### DISCUSSION TITLE
Adjust double click speed

### DISCUSSION THREAD

**Discussion by slipperyfrank** [hyprwm/Hyprland#3505] [General]
Does anyone know if there is a way to adjust the double click speed?
I would like to set a shorter double click timeout, but I can see how some users may also want the ability to set a slower one for accessibility reasons.
I am not sure where to look for answers, I have done a lot of searching for "[wayland | libinput | wlroots | hyprland] double click speed" on different search engines but with no results. Anyone here with experience on this?

**Comment by Kagukara** [comment]
This would be really nice to know as well. Changing `double-click` in `org.gnome.desktop.peripherals.mouse` doesn't seem to do anything. 

Having this option would delay the need to open up my mouse and re-solder new mouse buttons due to the switches starting to double click.
